### ISSUE TITLE
setBing wallpaper doesn't load on xfce lubuntu desktop

### ISSUE THREAD

**Issue by gastontancredi** [ypingcn/BingWallpaper#16]
[setBing wallpaper doesn't load on xfce lubuntu desktop]

ERROR
------------------------------------------------------
Something wrong 
Please check/adjust your setting or network connection, or just report this issue to maintainer with the following log.
------------------------------------------------------
sh: 1: xfconf-query: not found
sh: 1: notify-send: not found

**Comment by gastontancredi** [comment]
@ypingcn could you please help me to fix this issue? Thank you!

**Comment by ypingcn** [comment]
try those command in your terminal
1. `sudo apt install libnotify-bin` 
2. `sudo apt install xfconf` 

it could be better if i know which version of lubuntu are you using

**Comment by gastontancredi** [comment]
It worked! Thank you so much @ypingcn 
For ref, my version is 0.14.1
Thank you

**Comment by gastontancredi** [comment]
@ypingcn I have this error related to bingwallpaper, not sure if you might know the issue, thank you! https://github.com/whizzzkid/bing-wallpapers-for-linux/issues/25#issue-642455950

**Comment by gastontancredi** [comment]
> It worked! Thank you so much @ypingcn
> For ref, my version is 0.14.1
> Thank you

Hey it seems that it did not work as expected. The desktop changed but it doesn't change automatically, any idea about why it's happening? My desktop is LXQT and it is not listed

**Comment by ypingcn** [comment]
did you choose "autoupdate Bingwallpaper when tool startup" option ?  it's a little confusing right now. you need to choose this option and proper interval at the same time.

![option](https://user-images.githubusercontent.com/15324188/85242664-3f549280-b472-11ea-8672-9e5e946ba24b.png)

LXQT is not yet supported now , but it looks like xfce works in your os.

**Comment by gastontancredi** [comment]
So, when I run your code the deskop changed but it is not changing any more. The autoupdate is set. 
The desktop is not xfce (I was confused) it is LXQT, it seems it is the reason it doesn't work. Do you know if there is any other option? I tryied Bingwall but it is not supported either. This is the only I miss from Ubuntu

**Comment by gastontancredi** [comment]
![screen01](https://user-images.githubusercontent.com/67171481/85309262-b8f77b00-b488-11ea-8753-27b11a5a4342.jpg)

**Comment by gastontancredi** [comment]
![screen](https://user-images.githubusercontent.com/67171481/85309343-d88ea380-b488-11ea-9baf-6ea5f36bcf04.jpg)

**Comment by ypingcn** [comment]
ok , I will download lubuntu 20.04 in my spare time to support LXQT.

**Comment by ypingcn** [comment]
@gastontancredi  it occurred to me that "lastest" option in update means use the same image in bing.com, if you use "latest"+"every 60 minute", it will set the same image in one day.

lastest means "latest"
a new version is on the way

**Comment by gastontancredi** [comment]
@ypingcn are you testing this in LxQT? I made this setup and restarted. However when I restart the screen doesn't change. I also executed setwallpaper after the reboot (because it does not open automatically) and went to Update but it didn't work either

**Comment by gastontancredi** [comment]
![screen02](https://user-images.githubusercontent.com/67171481/85732693-f733ab00-b6d1-11ea-9622-4dc3ca4d8225.jpg)

**Comment by gastontancredi** [comment]
![screen03](https://user-images.githubusercontent.com/67171481/85732874-192d2d80-b6d2-11ea-9b77-b750ca2a9d2b.jpg)

**Comment by ypingcn** [comment]
can you paste what you get from [https://www.bing.com/HPImageArchive.aspx?format=js&idx=0&n=1](https://www.bing.com/HPImageArchive.aspx?format=js&idx=0&n=1) ??

**Comment by gastontancredi** [comment]
I get this:
{"images":[{"startdate":"20200625","fullstartdate":"202006250700","enddate":"20200626","url":"/th?id=OHR.GorchFock_ROW9740450416_1920x1080.jpg&rf=LaDigue_1920x1080.jpg&pid=hp","urlbase":"/th?id=OHR.GorchFock_ROW9740450416","copyright":"German navy tall ship Gorch Fock in waters close to Reykjavík, Iceland (© DEEPOL by plainpicture/Henn Photography)","copyrightlink":"javascript:void(0)","title":"","quiz":"/search?q=Bing+homepage+quiz&filters=WQOskey:%22HPQuiz_20200625_GorchFock%22&FORM=HPQUIZ","wp":true,"hsh":"5fe866a56fff557a84ca5a13ef0beb8c","drk":1,"top":1,"bot":1,"hs":[]}],"tooltips":{"loading":"Loading...","previous":"Previous image","next":"Next image","walle":"This image is not available to download as wallpaper.","walls":"Download this image. Use of this image is restricted to wallpaper only."}}


Yes I understand, thanks for trying anyway. Let me know if you realize how to make  it work, thank you! :)

**Comment by ypingcn** [comment]
can you try this command in your terminal ? it works in my vm . if it works in your os , i will make a new version in few days.
[  /home/lubuntu/BingWallpaper/bingwallpaper.jpg  ] should be replaced by the image path you want

```pcmanfm-qt --set-wallpaper=/home/lubuntu/BingWallpaper/bingwallpaper.jpg --wallpaper-mode=fit```

**Comment by gastontancredi** [comment]
The wallpaper changed into a black page

**Comment by gastontancredi** [comment]
![screen04](https://user-images.githubusercontent.com/67171481/85753958-138c1380-b6e3-11ea-954e-9f1546cb3e50.jpg)

**Comment by ypingcn** [comment]
oops... you should choose a image file in your computer first. ```--set-wallpaper``` should have a valid location.

```pcmanfm-qt --set-wallpaper=[your-image-file-path] --wallpaper-mode=fit```

**Comment by gastontancredi** [comment]
But I had an image with a valid location, look, I set one image and after I run the code it changes to black

**Comment by gastontancredi** [comment]
![screen05](https://user-images.githubusercontent.com/67171481/85755992-d0cb3b00-b6e4-11ea-8dc4-025b6f37015f.jpg)
![screen06](https://user-images.githubusercontent.com/67171481/85756000-d1fc6800-b6e4-11ea-914b-a787606dd8d7.jpg)
![screen07](https://user-images.githubusercontent.com/67171481/85756008-d32d9500-b6e4-11ea-89de-fc13ced99f0d.jpg)

**Comment by ypingcn** [comment]
oops .. you still don't catch my point. you should run this command in your terminal.

```pcmanfm-qt --set-wallpaper=/home/gaston/BingWallpaper/screen03.jpg --wallpaper-mode=fit```

if the wallpaper change from black page to screen03.jpg, it works .

**Comment by gastontancredi** [comment]
yeah so I reboot and I see it worked but this is not the wallpaper of the day, is the image I had in my computer, how can I make it change to the wallpaper of the day?

**Comment by ypingcn** [comment]
a new version is on the way. I will notify you when new version is ready.

**Comment by gastontancredi** [comment]
Thank you!! :)

**Comment by ypingcn** [comment]
new version is available , visit [https://github.com/ypingcn/BingWallpaper/releases/tag/1.0](https://github.com/ypingcn/BingWallpaper/releases/tag/1.0) to get deb package to update

**Comment by gastontancredi** [comment]
I've just installed this version and the desktop changed automatically by the desktop of the day. I will let you know if I see any bug in the upcoming days but so far so good! Thank you so much!

**Comment by gastontancredi** [comment]
@ypingcn Hey there is a weird bug here in my LxQt version. 
The wallpaper does not change automatically. It sends me a notification that it has been updated and downloaded in the respective folder. However what happens is that it downloads the same  old wallpaper, duplicating it.
I went to the folder that contains the wallpaper and I saw that it downloaded the same background twice with different dates.
I deleted all of them and hit "update" and then, the new one was downloaded and updated accordinly

**Comment by gastontancredi** [comment]
![screen](https://user-images.githubusercontent.com/67171481/86062519-41ed5400-ba3f-11ea-9a2d-f27cb5ad9a65.jpg)
1- You can see I had the bridge wallpaper the day June 29th. This one is the bing wallpaper from June 28 so I expected it to be updated per the "lavender" wallpaper. I received a notification that the Lavender background was downloaded automatically, however the wallpaper remained the same

**Comment by gastontancredi** [comment]
![screen01](https://user-images.githubusercontent.com/67171481/86062605-795c0080-ba3f-11ea-9984-1f24ca6de87e.jpg)
I checked the folder Bingwallpaper and I observed the same old bridge wallpaper downloaded twice with different names. One of them has the date of today that should be lavender and not the bridge

**Comment by gastontancredi** [comment]
![screen02](https://user-images.githubusercontent.com/67171481/86062665-9c86b000-ba3f-11ea-99b3-0e27e3b9773f.jpg)
I deleted all the images from the folder and I hit update

**Comment by gastontancredi** [comment]
![screen03](https://user-images.githubusercontent.com/67171481/86062693-ab6d6280-ba3f-11ea-921f-87233c5f7f60.jpg)
The lavender wallpaper was downloaded and updated properly. You can see that the name of the file has the date of today
